enterprise/app/services/captain/llm/conversation_faq_service.rb

```diff
@@ -106,6 +107,11 @@ def chat_parameters
     }
   end
 
+  def account_language
+    account_locale = @conversation.account.locale
+    ISO_639.find(account_locale)&.english_name&.downcase || 'english'
+  end
```

Comment:
seems repeated every where I would suggest adding a method in account to simplify this. @sojan-official 

Something like account.english_locale

Reply:
Done, added as `locale_english_name`